src/pages/academy/components/video-banner/carousel/_carousel-style.ts

```diff
@@ -86,11 +86,10 @@ export const StyledChevron = styled((props) => <Chevron {...props} />)`
 
     path {
         ${(props) => {
-            const black_color = 'fill: var(--color-black);'
-            const red_color = 'fill: var(--color-red);'
+            const choosed_color = `fill: var(--color-${props.color});`
             const default_color = 'fill: var(--color-white);'
 
-            return props.black ? black_color : props.red ? red_color : default_color
+            return props.color !== 'custom' ? choosed_color : default_color
```

Comment:
```suggestion
            return props.color === 'custom' ? default_color : chosen_color
```
* Negatives are just a bit harder to understand than positives. So, when possible, conditionals should be expressed as positives 🙂